lib/mycha_installer.coffee

```diff
@@ -0,0 +1,67 @@
+async = require 'async'
+{exec} = require 'child_process'
+fsExtra = require 'fs-extra'
+path = require 'path'
+
+
+class MychaInstaller
+
+  constructor: ({@cwd, @testHelperPath}) ->
+
+
+  install: (done) ->
+    async.parallel [
```

Comment:
updated to series, are you suggesting the output also includes what files are written.
That may look weird after the `npm install`. The current output is:
```
sinon-chai@2.7.0 ../../../../var/folders/yn/rz7n_fdn2vq5_rdmvn06yn500000gn/T/tmp-4827gEMsTw2ADKOx/node_modules/sinon-chai

chai@2.1.2 ../../../../var/folders/yn/rz7n_fdn2vq5_rdmvn06yn500000gn/T/tmp-4827gEMsTw2ADKOx/node_modules/chai
├── assertion-error@1.0.0
└── deep-eql@0.1.3 (type-detect@0.1.1)

sinon@1.14.1 ../../../../var/folders/yn/rz7n_fdn2vq5_rdmvn06yn500000gn/T/tmp-4827gEMsTw2ADKOx/node_modules/sinon
├── lolex@1.1.0
├── util@0.10.3 (inherits@2.0.1)
└── formatio@1.1.1 (samsam@1.1.2)
```